archive/html/security/Ai-security-best-practices.html

```
@@ -0,0 +1,2245 @@
+        .sidebar.open {
+          left: 0;
+        }
+        main.content {
+          padding: 70px 20px 80px;
+          max-width: 100%;
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <button id="menu-toggle" class="menu-toggle" aria-label="メニューを開く">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)